archinstall/lib/installer.py

```diff
@@ -1506,7 +1506,7 @@ def _add_efistub_bootloader(
 
 		parent_dev_path = device_handler.get_parent_device_path(boot_partition.safe_dev_path)
 
-		cmd_template = (
+		cmd_template = [
```

Comment:
Why was this changed? What did this have to do with this pull request?

This was a tuple because it avoids unnecessary mutability, has the slight benefit of using less memory, and communicates that it is a static sequence meant to be reused. It is being formatted and expanded into lists later anyway.

Reply:
Thanks for pointing this out, it was accidentally committed, reverted https://github.com/archlinux/archinstall/pull/3648